### PR TITLE
Merge gen-payloads into one

### PR DESCRIPTION
**Pull request by jupierce**
Depends on https://github.com/openshift/doozer/pull/144

### CODE REVIEW

File: jobs/build/build-sync/build.groovy [hunk 1]
```diff
@@ -97,61 +92,32 @@ release:gen-multiarch-payload
...
-    } else {
+    for ( String arch: arches ) {
         retry ( 3 ) {
+            echo("Attempting to mirror arch: ${arch}")
+            // Always login again. It may expire between loops
+            // depending on amount of time elapsed
             buildlib.registry_quay_dev_login()
-            buildlib.oc "${logLevel} image mirror ${dryRun} --filename=${ocMirrorInput}"
-            currentBuild.description += "\nMirrored: default arch (amd64)"
+            buildlib.oc "${logLevel} image mirror ${dryRun} --filename=${mirrorWorking}/src_dest.${arch}"
```
Comment (vfreex) [inline jobs/build/build-sync/build.groovy]: Is `--filter-by-os` missing or something related to the command changed?

Comment (jupierce) [reply jobs/build/build-sync/build.groovy]: The entries in the file are already arch specific at this point. There should be nothing to filter.

**Comment by vfreex** [comment]
@jupierce PR needs rebase.

**Comment by jupierce** [comment]
Merging since https://github.com/openshift/doozer/pull/144 merged.

**Pull request merged**